ackermann_steering_controller/package.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>ackermann_steering_controller</name>
+  <version>0.1.0</version>
+  <description>Controller for a steer drive mobile base.</description>
+  <maintainer email="p595201m@mail.kyutech.jp">Masaru Morita</maintainer>
+
+  <license>BSD</license>
+
+  <url type="bugtracker">https://github.com/ros-controls/ros_controllers/issues</url>
+  <url type="repository">https://github.com/ros-controls/ros_controllers/ackermann_steering_controller</url>
+  <author email="p595201m@mail.kyutech.jp">Masaru Morita</author>
+
+  <buildtool_depend>catkin</buildtool_depend>
+
+  <depend>boost</depend>
+  <depend>controller_interface</depend>
+  <depend>diff_drive_controller</depend>
+  <depend>hardware_interface</depend>
+  <depend>nav_msgs</depend>
+  <depend>realtime_tools</depend>
+  <depend>roscpp</depend>
+  <depend>tf</depend>
+
+  <build_depend>pluginlib</build_depend>
```

Comment:
<exec_depend> on pluginlib and urdf as well.
(They might provided shared libraries that we link against..)

Reply:
Fixed at https://github.com/ros-controls/ros_controllers/pull/356/commits/957b22f3928e183d567db30de70a08d0575cfa05